submissions/egwoloOkorodudu/css/contact.css

```diff
@@ -0,0 +1,270 @@
+*{
+    box-sizing: border-box;
+}
+body{
+    margin: 0;
+    font-family: 'Open Sans', sans-seri;
+    line-height: 1.5;
+    height: 100%;
+}
+/*---header section----*/
+.header{
+    overflow: hidden;
+    background-color: #b6249e;
+    padding: 25px 4em 0 20px;
```

Comment:
Use the same unit all through @Egwolo2

Reply:
Okat @ToluAkin 
Noted

Reply:
Corrections has been made @ToluAkin  and @adaugochi 
Thanks a lot